src/functionalTest/java/functions/XMLFunctions.java

```diff
@@ -0,0 +1,23 @@
+package functions;
```

Comment:
Generally speaking, the package structure should mirror that of the main source classes.  Store the functional test classes under src/functionalTest/java/uk/gov/moj/sdt so that the package for this class becomes uk.gov.moj.sdt.functions.

The same comment applies to all the other classes.